php/Dockerfile

```diff
@@ -5,11 +5,14 @@ LABEL maintainer="eric.poe@gmail.com"
 # Set TERM environment var - so terminal cmds like `top` work
 ENV TERM=xterm
 
-RUN yum install -y wget epel-release
-RUN wget http://rpms.famillecollet.com/enterprise/remi-release-6.rpm
-RUN rpm -Uvh remi-release-6*.rpm
-RUN yum update -y
-RUN yum --enablerepo=remi-php71 install -y \
+RUN yum install -y wget epel-release \
+        http://rpms.famillecollet.com/enterprise/remi-release-6.rpm \
```

Comment:
I believe the only usage of `wget` in this and child images was to wget the remi repo. Since you have simplified the download and install by not using `wget` for that repo, `wget` can be safely removed from the install.

Reply:
I thought the same thing, I just wasn't sure if something else needed it, like PHP. But PHP uses Curl now that I think of it. I'll pull it out and we can see.

Reply:
https://github.com/Stowers-LIMS/docker-php-base/pull/3/commits/24550aa1b4a97bca0882b3b6928e4b1e9ab5f6ed